arming.h

```diff
@@ -0,0 +1,50 @@
+#ifndef ARMING_H
+#define	ARMING_H
+
+#ifdef	__cplusplus
+extern "C" {
+#endif
+ 
+#include <xc.h>
+
+// Time between main loop code execution
+#define MAX_LOOP_TIME_DIFF_ms 1000
+    
+// LEDs
+#define RED_LED_ON() (LATC7 = 0)
+#define RED_LED_OFF() (LATC7 = 1)
+#define BLUE_LED_ON() (LATC6 = 0)
+#define BLUE_LED_OFF() (LATC6 = 1)
+#define WHITE_LED_ON() (LATC5 = 0)
+#define WHITE_LED_OFF() (LATC5 = 1)
+    
+// Buzzer
+#define BUZZER_ON() (LATB4 = 1)
+#define BUZZER_OFF() (LATB4 = 0)
+
+// Altimeters
+#define ARM_A1() (LATC3 = 1)
+#define DISARM_A1() (LATC3 = 0)
+#define ARM_A2() (LATC2 = 1)
+#define DISARM_A2() (LATC2 = 0)
+    
+// Voltage Monitoring
+#define BATTERY_1_PIN   channel_ANA0
+#define BATTERY_2_PIN    channel_ANA1
+#define A1_MAIN_PIN     channel_ANB3
+#define A1_DROGUE_PIN   channel_ANB2
+#define A2_MAIN_PIN     channel_ANB1
+#define A2_DROGUE_PIN   channel_ANB0
```

Comment:
When you configure the analog channels, you can actually give them meaningful names, saving yourself the extra round of `#define`s. So then your channels could be called something like `channel_A1_MAIN` or whatever.

Eg. from injector: https://github.com/waterloo-rocketry/cansw_injector/blob/aa1d5e417303065f9a181f73eedb278ddf167348/mcc_generated_files/adcc.h#L89